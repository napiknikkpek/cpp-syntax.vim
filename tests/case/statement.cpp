while (true)
  ;
for (auto i : {})
  ;
for (auto [i, v] : map)
  ;
for (auto& [i, v] : map | value)
  ;
for (const auto& [a, b, c] : map.foo())
  ;
