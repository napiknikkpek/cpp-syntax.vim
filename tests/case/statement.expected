aaaaa..cccc.
...
aaa..aaaa.x......
...
aaa..aaaa..x..x....xxx.
...
aaa..aaaa...x..x....xxx...xxxxx.
...
aaa..aaaaa.aaaa...x..x..x....xxx.fff...
...
